shoplist/src/main/java/com/netguru/codereview/network/model/ShopListItemResponse.kt

```diff
@@ -0,0 +1,10 @@
+package com.netguru.codereview.network.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+data class ShopListItemResponse(
```

Comment:
using data classes in kotlin is helpful as it minimizes the code, it's a good approach